File: sakai-ng-master/src/app/Models/EmpleadoViewModel.ts
export class Empleado{
    ciu_Id?:String;
    est_Descripcion?:String;
    crg_Descripcion?:String;
    empl_Id?:String;
    empl_Nombre?:String;
    empl_Apellido?:String;
    empl_Sexo?:String;    
    empl_FechaNac?:String;
    //empl_Correo:string;

    empl_DNI?:String;
    ciu_Descripcion:string;

}

export class dropEmpleado{
    value?:String;
    text?:String;
}

export class EmpleadoEnviar {
    

    Empl_DNI?:String;
    Empl_Id:string;
    Empl_Nombre: string;
    Empl_Apellido: string;
    Empl_FechaNac: string;
    Empl_Sexo: string;
    Ciu_Id: string;
    Est_ID: string;
    Carg_Id: string;
    //Sucu_Id: string;
    //Empl_Correo:string;


}

export class Fill {
    empl_Id: string;
    empl_Nombre: string;
    empl_Apellido?:String;
    empl_FechaNac?:String;
    empl_Sexo?:String;
    ciu_Id?:String;
    est_ID?:String;
    carg_Id:string;
    crg_Descripcion?:string;
    est_Descripcion?:String;
    ciu_Descripcion?:String;
    //empl_Correo:string;
    empl_DNI?:String;
    dep_Id?:String;
    dep_Descripcion?:String;
    usuarioCreacion: string;
    usuarioModificacion: string;
    fechaCreacion : string;
    fechaModificacion : string;
}

File: Grupo_Rac/Grupo_Rac.Common/Models/EmpleadoViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Grupo_Rac.Common.Models
{
    public class EmpleadoViewModel
    {
        [NotMapped]
        public string Empl_DNI { get; set; }
        public int Empl_Id { get; set; }
        public string Empl_Nombre { get; set; }
        public string Empl_Apellido { get; set; }
        public string Empl_Sexo { get; set; }
        public DateTime Empl_FechaNac { get; set; }
        public string Ciu_Id { get; set; }
        public int Est_ID { get; set; }
        public int Carg_Id { get; set; }
        public int Sucu_Id { get; set; }
        public int? Empl_UsuarioCreacion { get; set; }
        public DateTime? Empl_FechaCreacion { get; set; }
        public int? Empl_UsuarioModificacion { get; set; }
        public DateTime? Empl_FechaModificacion { get; set; }
        public bool? Empl_Estado { get; set; }

        [NotMapped]
        public string UsuarioCreacion { get; set; }

        [NotMapped]
        public string UsuarioModificacion { get; set; }

        [NotMapped]
        public string Ciu_Descripcion { get; set; }
        [NotMapped]
        public string Est_Descripcion { get; set; }

    }
}
